components/about/TeamMember.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="team-member">
+    <v-container>
+      <v-row>
+        <v-col sm="4">
+          <v-avatar size="170" color="grey">
+            <img
+            v-if="customPic"
+            :src="require(`~/assets/img/profile/${customPic}`)"
+            :alt="githubHandle"
+            style="object-fit: cover"
+            />
+            <img v-else :src="githubAvatarUrl" :alt="githubHandle" />
+          </v-avatar>
+        </v-col>
+        <v-col sm="8">
+          <div class="team-member-detail">
+            <div class="team-member-detail-content">
+              <div class="team-member-detail-info">
+              <h3>{{name}}</h3>
+              <h4>{{role}}</h4>
+              </div>
+              <div class="team-member-detail-actions">
+                <v-btn icon link color="white" :href="githubProfileUrl">
+                    <v-icon>mdi-github</v-icon>
+                  </v-btn>
+                  <v-btn v-if="twitterHandle" icon link color="white" :href="twitterProfileUrl">
+                    <v-icon>mdi-twitter</v-icon>
+                </v-btn>
+              </div>
+            </div>
+          </div>
+        </v-col>
+      </v-row>
+    </v-container>
+  </div>
+</template>
+<script>
+const getGithubProfileUrl = function (handle) {
+  return `https://github.com/${handle}`;
+};
+
+export default {
+  props: {
+    githubHandle: {
+      type: String,
+      required: true,
+    },
+    name: {
+      type: String,
+      required: true,
+    },
+    role: {
+      type: String,
+      required: true,
+    },
+    twitterHandle: String,
+    customPic: String,
+  },
+  computed: {
+    githubAvatarUrl: function () {
+      return `${getGithubProfileUrl(this.githubHandle)}.png`;
+    },
+    githubProfileUrl: function () {
+      return getGithubProfileUrl(this.githubHandle);
```

Comment:
nit: not sure it's worth making a method for this `get` method for only two spots and then not for twitter version. 😄 
NBD, just pointing it out

Reply:
> It’s looking great so far! I noticed a small issue while swiping on my phone to see the tweets. The page extends further right than intended. Maybe this is due to the large image at the top?
> 

Thanks @theyokohamalife !

So this issue is caused by page overflow, and I'd noticed it before (it happens on our current page too) but had forgotten about it. If we disable horizontal scroll/overflow on the root element for this it will fix.

Reply:
@theyokohamalife fixed by adding global overflow-x limitations.